packages/wonder-blocks-popover/src/components/__docs__/popover-content-core.stories.js

```diff
@@ -0,0 +1,186 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Color from "@khanacademy/wonder-blocks-color";
+import Clickable from "@khanacademy/wonder-blocks-clickable";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {
+    Body,
+    HeadingSmall,
+    LabelLarge,
+} from "@khanacademy/wonder-blocks-typography";
+
+import type {StoryComponentType} from "@storybook/react";
+import {PopoverContentCore} from "@khanacademy/wonder-blocks-popover";
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+
+// NOTE: We are reusing an existing Cell SB Story to test how Popovers can be
+// composed by Cells.
+// eslint-disable-next-line monorepo/no-relative-import
+import {ClickableDetailCell} from "../../../../wonder-blocks-cell/src/components/__docs__/detail-cell.stories.js";
+
+export default {
+    title: "Popover/PopoverContentCore",
+    component: PopoverContentCore,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        docs: {
+            description: {
+                component: null,
+            },
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+    },
+    decorators: [
+        (Story: any): React.Element<typeof View> => (
+            <View style={styles.example}>{Story()}</View>
+        ),
+    ],
```

Comment:
How do you feel about adding an a small inline `argTypes` here and then setting the control type for children to `null`? I usually do this because I find the giant json object under the control panel for React component children to be unreadable and unhelpful.

![Screen Shot 2022-08-29 at 2 52 33 PM](https://user-images.githubusercontent.com/13231763/187305638-a09ec958-8cc7-427d-9f31-4da6fb004286.png)